src/marqo/api/configs.py

```diff
@@ -31,6 +31,7 @@ def default_env_vars() -> dict:
         EnvVars.MARQO_LOG_LEVEL: "info",
         EnvVars.MARQO_MEDIA_DOWNLOAD_THREAD_COUNT_PER_REQUEST: 5,
         EnvVars.MARQO_IMAGE_DOWNLOAD_THREAD_COUNT_PER_REQUEST: 20,
+        EnvVars.MARQO_MAX_VIDEO_AUDIO_SEARCH_FILE_SIZE: 387973120,  # 370 megabytes in bytes
```

Comment:
I thought we use 100MB?

Reply:
100 was the old limit on add docs and 20s for search/embed.

370 is about how much we can download within 20s. This is to avoid regression in capability